.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -0,0 +1,64 @@
+name: Bug Report
+description: Create a bug report to help us improve
+title: "[Descriptive title] "
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **NOTE: General questions should go to the [Discord chat](https://discord.com/invite/xSAwrnCWcg) instead of the issue tracker.**
```

The discord invite link redirects to gossamer-general channel

```suggestion
        **NOTE: General questions should go to the [Discord chat](https://discord.gg/aMxzVcr) instead of the issue tracker.**
```